lib/rules/lines-around-comment.js

```diff
@@ -93,6 +93,12 @@ module.exports = {
                     },
                     allowArrayEnd: {
                         type: "boolean"
+                    },
+                    ignorePattern: {
+                        type: "string"
+                    },
+                    applyDefaultPatterns: {
```

Comment:
If we're putting the deprecation in a different PR, should we wait to merge this one until the other one is in? I don't want to create an option in this PR and then immediately deprecate/rename it.